magento/pwa-studio
# feat: Simplify image middleware with express-sharp update

## Description

The express-sharp library has updated to v3.0.0, which includes a [breaking change](https://github.com/magento-research/express-sharp/pull/5) to the URL scheme, and therefore the public API of the library.

This helps us simplify the code in addImgOptMiddleware, because we don't have to rewrite the URL as aggressively--it's closer to the Fastly paradigm.

## Related Issue





Closes #1808 and #1842 

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Visit #1808 and follow those verification steps.
2. Check out this branch and reproduce the error conditions.
3. Should work!
4. Bug #1842 should also be resolved.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
